src/indigo/less/buttons.less

```diff
@@ -66,4 +66,9 @@ button {
       }
     }
   }
+}
+
+//TrashIndex
+.btn-hide {
+  display: none !important;
```

Comment:
tohle zustalo ze stylu z TrashIndex pouzivany. Izoloval jsem a prehodil sem k buttonum, takze je to univerzalne pouzitelny bez kontextu. Predim bylo nastavano visibility: hidden, prehodil jsem to na display:none, jinak tam byla misto buttonu byla dira